automation_tools/__init__.py

```diff
@@ -1154,40 +1170,86 @@ def client_registration_test(clean_beaker=True, update_packages=True):
 
     # Register and subscribe
     print('Register/Subscribe using Subscription-manager.')
-    run('subscription-manager register --force'
-        ' --org="{0}"'
-        ' --activationkey="{1}"'
-        ''.format(org, act_key))
+    cmd = ('subscription-manager register --force'
+           ' --org="{0}"'
+           ' --activationkey="{1}"')
+    if rel_ver:
+        cmd = cmd + ' --release="{2}"'
+        run(cmd.format(org, act_key, rel_ver))
+    else:
+        run(cmd.format(org, act_key))
     print('Refreshing Subscription-manager.')
     run('subscription-manager refresh')
     print('Performing yum clean up.')
     run('yum clean all', quiet=True)
+
+    # Checking Package installation
     print('"Firefox" and "Telnet" should not be installed.')
     run('rpm -q firefox telnet', warn_only=True)
-    print('Installing "Firefox" and "Telnet".')
-    run('yum install -y firefox telnet', quiet=True)
-    print('"Firefox" and "Telnet" should be installed.')
-    run('rpm -q firefox telnet')
-    print('Removing "Firefox" and "Telnet".')
-    run('yum remove -y firefox telnet', quiet=True)
-    print('Checking if "Firefox" and "Telnet" are installed.')
-    run('rpm -q firefox telnet', warn_only=True)
-    print('Installing "Web Server" group.')
-    run('yum groupinstall -y "Web Server"', quiet=True)
-    print('Checking for "httpd" and starting it.')
-    run('rpm -q httpd')
-    manage_daemon('start', 'httpd', warn_only=True)
-    print('Stopping "httpd" service and remove "Web Server" group.')
-    manage_daemon('stop', 'httpd', warn_only=True)
-    run('yum groupremove -y "Web Server"', quiet=True)
-    print('Checking if "httpd" is really removed.')
-    run('rpm -q httpd', warn_only=True)
+    print('Installing "Telnet".')
+    run('yum install -y telnet', quiet=True)
+    print('"Telnet" should be installed.')
```

Comment:
You can check if telnet was installed or not like:

```python
result = run('yum install -y telnet', quiet=True)
if result.succeeded:
    print('"Telnet" is installed.')
```

For more info http://docs.fabfile.org/en/1.10/api/core/operations.html?highlight=succeeded#fabric.operations.run